.azure/pipelines/helix-test.yml

```diff
@@ -23,3 +23,19 @@ jobs:
     - name: Helix_logs
       path: artifacts/log/
       publishOnError: true
+
+- template: jobs/default-build.yml
+  parameters:
+    jobName: Helix_Arm
+    jobDisplayName: 'Tests: Helix Arm'
+    agentOs: Windows
+    timeoutInMinutes: 240
+    steps:
+    - script: .\build.cmd -arch arm -all -ci /t:Helix /p:IsHelixJob=true /bl:$env:BUILD_SOURCESDIRECTORY\artifacts\logs\SendToHelixArm.binlog
```

Comment:
```suggestion
    - script: .\build.cmd -arch arm64 -all -ci /t:Helix /p:IsHelixJob=true /bl:$env:BUILD_SOURCESDIRECTORY\artifacts\logs\SendToHelixArm.binlog
```

Reply:
This is the error from trying arm64: Cannot validate argument on parameter 'Architecture'. The argument "arm64" does not belong to the set "x64,x86,arm"

Reply:
Cc @JunTaoLuo

I don't think we actually support building for Windows Arm64 yet, only Linux Arm64. That's why you're getting this error. Maybe this should build on Linux instead?

Reply:
Correct we don't have win-arm64 yet.